pallets/aggregate/src/lib.rs

```diff
@@ -340,9 +346,15 @@ pub mod pallet {
     /// Shortcut to get the Domain type from config.
     #[derive(Encode, Decode, TypeInfo, MaxEncodedLen)]
     #[scale_info(skip_type_params(T))]
-    pub(crate) struct Domain<T: Config>(DomainType<T>);
-
-    impl<T: Config> Domain<T> {
+    pub struct Domain<T: Config>(DomainType<T>);
+
+    impl<T: Config> Domain<T>
+    where
+        T::AccountId: MaxEncodedLen,
+        BalanceOf<T>: MaxEncodedLen,
+        T::RuntimeHoldReason: MaxEncodedLen,
```

Comment:
You call remove all bounds here: are use less

Reply:
yes indeed they are useless now